libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Base.hs

```diff
@@ -31,159 +31,59 @@ import Cardano.Ledger.Conway.Governance (
   dStateSpec,
   delegCertSpec,
   delegEnvSpec,
   epochEnvSpec,
   epochSignalSpec,
   epochStateSpec,
-  genProposalsSplit,
   govCertEnvSpec,
   govCertSpec,
-  govEnvSpec,
-  govProceduresSpec,
-  govProposalsSpec,
   newEpochStateSpec,
   utxoEnvSpec,
   utxoStateSpec,
   utxoTxSpec,
   vStateSpec,
  )
 import Test.Cardano.Ledger.Constrained.Conway.Instances ()
-import Test.Cardano.Ledger.Conway.ImpTest (impAnn, logEntry)
+import Test.Cardano.Ledger.Conway.Arbitrary ()
 import Test.Cardano.Ledger.Imp.Common hiding (arbitrary, forAll, prop)
 
-data ConwayGovTransContext era
-  = ConwayGovTransContext
-      (EnactState era)
-      (TxId (EraCrypto era))
-      (Proposals era)
-  deriving (Generic)
-
-deriving instance EraPParams era => Eq (ConwayGovTransContext era)
-
-deriving instance EraPParams era => Show (ConwayGovTransContext era)
-
-instance
-  ( Era era
-  , ToExpr (PParamsHKD Identity era)
-  , ToExpr (PParamsHKD StrictMaybe era)
-  ) =>
-  ToExpr (ConwayGovTransContext era)
-
-instance HasSimpleRep (ConwayGovTransContext era)
-
-instance
-  ( IsConwayUniv fn
-  , EraPParams era
-  , HasSpec fn (EnactState era)
-  , HasSpec fn (Proposals era)
-  ) =>
-  HasSpec fn (ConwayGovTransContext era)
-
-instance Inject (ConwayGovTransContext era) (EnactState era) where
-  inject (ConwayGovTransContext x _ _) = x
-
-instance EraCrypto era ~ c => Inject (ConwayGovTransContext era) (TxId c) where
-  inject (ConwayGovTransContext _ x _) = x
-
-instance EraCrypto era ~ c => Inject (ConwayGovTransContext era) (Proposals era) where
-  inject (ConwayGovTransContext _ _ x) = x
-
-instance
-  ( NFData (SpecRep (ConwayGovPredFailure Conway))
-  , IsConwayUniv fn
-  ) =>
-  ExecSpecRule fn "GOV" Conway
-  where
-  type ExecContext fn "GOV" Conway = (TxId StandardCrypto, ProposalsSplit)
-
-  environmentSpec _ = govEnvSpec
-
-  stateSpec _ = govProposalsSpec
-
-  signalSpec _ = govProceduresSpec
-
-  genExecContext = (,) <$> arbitrary <*> genProposalsSplit 20
-
-  runAgdaRule env st sig =
-    first (\e -> OpaqueErrorString (T.unpack e) NE.:| [])
-      . computationResultToEither
-      $ Agda.govStep env st sig
-
-  translateInputs env st sig (txId, _) = do
-    let
-      modifiedEnactState =
-        def
-          & ensPrevGovActionIdsL .~ toPrevGovActionIds (st ^. pRootsL)
-      modifiedCtx = ConwayGovTransContext modifiedEnactState txId st
-    logEntry $ "modifiedCtx:\n" <> showExpr modifiedCtx
-    agdaEnv <-
-      impAnn "Translating the environment" . expectRight . runSpecTransM modifiedCtx $ toSpecRep env
-    logEntry $ "agdaEnv:\n" <> showExpr agdaEnv
-    agdaSt <- impAnn "Translating the state" . expectRight . runSpecTransM modifiedCtx $ toSpecRep st
-    logEntry $ "agdaSt:\n" <> showExpr agdaSt
-    agdaSig <- impAnn "Translating the signal" . expectRight . runSpecTransM modifiedCtx $ toSpecRep sig
-    logEntry $ "agdaSig:\n" <> showExpr agdaSig
-    pure (agdaEnv, agdaSt, agdaSig)
-
-  testConformance ctx env st sig = property $ do
```

Comment:
This was not moved to `ExecSpecRule.Conway.Gov` (unless I'm practically blind), so we are basically removing it. Why?

Reply:
This is just overriding the default implementation, and we don't need to do that. I think that this overriding was a left-over from an experiment.